packages/multicast_dns/lib/src/resource_record.dart

```diff
@@ -0,0 +1,182 @@
+// Copyright 2018 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:meta/meta.dart';
+
+/// Enumeration of support resource record types.
+class RRType {
+  /// An IPv4 Address record (1).
+  static const int a = 1;
+
+  /// An IPv6 Address record (28).
+  static const int aaaa = 28;
+
+  /// An IP Address reverse map record (12).
+  static const int ptr = 12;
+
+  /// An available service record (33).
+  static const int srv = 33;
+
+  /// A text record (16).
+  static const int txt = 16;
+
+  // TODO(dnfield): Can we support ANY in some meaningful way?
```

Comment:
nit: per our style guide (iirc), avoid TODOs that are questions. We should just say something like `TODO(...): support ANY in some meaningful way` or not bother with it.